Documentation-rendertest/Typesetting/Index.rst

```diff
@@ -1,170 +1,132 @@
 .. include:: /Includes.rst.txt
 
-==============
+..  _typesetting:
+
+===========
 Typesetting
-==============
+===========
 
-.. contents:: This page
-   :backlinks: top
-   :class: compact-list
-   :local:
+..  contents:: Table of contents
 
+Introduction
+============
 
+This is a typesetting test in reStructuredText. You will be required to demonstrate various formatting options including:
 
-==============
-Typesetting H1
-==============
+- **Headers**
+- *Emphasis*
+- `Inline code`
+- Lists
+- Blockquotes
+- Tables
 
-Typesetting H2
+..  _headers:
+
+Headers
+=======
+
+There are multiple levels of headers available:
+
+Level 2 Header
 ==============
 
-Typesetting H3
+Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod
+tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.
+
+Level 3 Header
 --------------
 
-Typesetting H4
+At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd
+gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.
+Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod
+tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.
+
+Level 4 Header
 ~~~~~~~~~~~~~~
 
-Typesetting H5
-++++++++++++++
+At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd
+gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.
 
-Typesetting H6
-##############
+..  _emphasis:
 
-==============
-Typesetting H1
-==============
+Emphasis
+========
 
-Just a line.
+Emphasizing text can be done using asterisks or underscores:
 
-Typesetting H2
-==============
+*This text is emphasized.* _This text is also emphasized._
 
-Just a line.
+Code
+====
 
-Typesetting H3
---------------
+Inline code
+-----------
 
-Just a line.
+You can highlight inline code using backticks: `some code`, :php:`$somePHP`.
```

Comment:
Please also add double backticks, like described here: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#inline-markup

Reply:
The two are actually transformed into the same node and rendered the same. There are pages that test different inline roles more extended